src/config-schema.ts

```diff
@@ -25,6 +25,12 @@ export const configSchema = {
     _type: Type.String,
     _default: "Pharmacy",
   },
+  medicationRequestExpirationPeriodInDays: {
+    _type: Type.Number,
+    _description:
+      "Medication Requests older that this will be considered expired",
+    _default: 90,
+  },
```

Comment:
New config parameter to configure the expiration period in days (which is compared against the encounter date)

Reply:
Does this mean logic not to allow expired requests has been added as well? And also how does this work in relation to recurring medications?

Reply:
Yes, right now you can't dispense orders older than x days old, (which a default of 90)... you are right that this will likely need to be tweaked based on refills, see my comment below.